### PR TITLE
Add a button to go to admin page from the root-admin

### PR DESCRIPTION
**Pull request by AlasDiablo**
Screenshot_20240611_134941

### CODE REVIEW

File: src/app/js/root-admin/Tenants.js [hunk 1]
```diff
@@ -378,12 +409,13 @@ const Tenants = ({ handleLogout }) => {
             flex: 1,
             renderCell: (params) => {
                 return (
-                    <Button
+                    <IconButton
```
Comment (parmentf) [inline src/app/js/root-admin/Tenants.js:412]: **question:** ‏What's the difference between `Button` and `IconButton` ?

File: src/app/js/root-admin/Tenants.js [hunk 2]
```diff
@@ -267,15 +270,43 @@ const Tenants = ({ handleLogout }) => {
...
+                                '&:hover': {
+                                    textDecoration: 'none',
+                                },
+                            }}
+                        >
+                            {params.value}
+                        </Link>
+                        <IconButton
+                            size="large"
+                            href={`${baseUrl}/instance/${name}/admin`}
```
Comment (parmentf) [inline src/app/js/root-admin/Tenants.js:303]: **praise:** ‏Good idea!

**Review by parmentf — APPROVED** [review]

File: src/app/js/root-admin/Tenants.js (hunk 1 above)
Comment (AlasDiablo) [reply src/app/js/root-admin/Tenants.js:412]: Is a variant of the Button when we have one icon as the children.
For example when you want a button with one of the following icon https://mui.com/material-ui/material-icons/

Comment (AlasDiablo) [reply src/app/js/root-admin/Tenants.js:412]: You can learn for here:
https://mui.com/material-ui/react-button/#icon-button

**Pull request merged**